### DISCUSSION TITLE
Can't login to Prestashop Back Office

### DISCUSSION THREAD

**Discussion by hcantave** [PrestaShop/PrestaShop#31536] [Q&A]
I need help to login in Prestashop Back Office.  I had Prestashop 1.7.7.5 . I even install 8.0 and still have the same problem. but can't access the BO to take care of the site. How can you help?

**Comment by kpodemski** [comment]
Hello @hcantave 

Maybe you could take a look for [support](https://www.prestashop-project.org/support/). It may be related to your server configuration if you can't log in to PrestaShop 1.7.7.5 or 8.0. You may have a PHP version incompatible with PrestaShop. Maybe there's a problem somewhere else. It's hard to tell without login to your server.
